src/service/FilterTodo.ts

```diff
@@ -0,0 +1,28 @@
+import { Todo } from '../types/Todo';
+import { Status } from '../types/Status';
+
+export const FilterTodo = (todos: Todo[], text: string, status: Status) => {
```

Comment:
it's neither a component nor class
```suggestion
export const filterTodos = (todos: Todo[], text: string, status: Status) => {
```